src/main/java/org/terasology/ligthandshadow/componentsystem/LASUtils.java

```diff
@@ -178,6 +178,14 @@ public static String getOppositionTeam(String team) {
         return null;
     }
 
+    public static float getYaw(String team) {
```

Comment:
`getYaw` seems good for a utility function, but doesn't match the hard-coded (and not documented) "assumptions" about _what_ yaw we're calculating here (relation to the teleport destinations of the teams, etc.)